src/components/create-hack.js

```diff
@@ -26,32 +28,28 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-function CreateHack() {
+function CreateHack(props) {
   const classes = useStyles();
 
   const [title, setTitle] = useState("");
   const [description, setDescription] = useState("");
+  const [goal, setGoal] = useState("");
   const [show, setShow] = useState(false);
 
-  const handleChange = event => {
-    if (event.target.name === "title") {
-      setTitle(event.target.value);
-    } else {
-      setDescription(event.target.value);
-    }
-  };
+  const history = props.history
 
   const handleSubmit = event => {
     event.preventDefault();
     setShow(true);
     const obj = {
       title: title,
       description: description,
-      goal: "temp"
+      goal: goal
     };
-    Axios.post(process.env.REACT_APP_API_URL+"addhack", obj).then(res => {
+    Axios.post(process.env.REACT_APP_API_URL + "addhack", obj).then(res => {
```

Comment:
Aware you didnt meaningfully change this line in this PR, but just want to raise it as something to think about.

The Robustness principle, which was originally intended for talking about networking protocols, but can apply as a good rule generally says:

> Be conservative in what you send; be liberal in what you accept.

This could be interpreted here as the environment variable you are accepting may be reasonably defined as something like

* `https://www.example.com`
* `https://www.example.com/`

The above code could end up being `https://www.example.comaddhack if the person configuring the environment is no aware of the implementation detail of how it is used.

Better practice would be to allow either form to be configured and account for it (you can either write a small function to do it if its simple enough, or something as complex as urls, might be better to use a library - https://www.npmjs.com/package/url-join)